lib/dal/src/pkg/import.rs

```diff
@@ -211,7 +215,7 @@ async fn import_change_set(
     // Code level feature flag to allow updating any schema by simpling installing a package with a schema of the same name
     // This existis mostly to help debugging
     // Should always be `false` in production
-    let update_even_if_not_builtin = false;
+    let update_even_if_not_builtin = true;
```

Comment:
Curious about this flag and the comment (which says it should always be false)

Reply:
You are right about the comment, I may change it.

Basically the original spec defined that only builtins would behave like that, but we realized that this was not ideal. I didn't want to remove that flag right now to allow us to quickly toggle the feature.